### DISCUSSION TITLE
Trouble with NES/Famicom

### DISCUSSION THREAD

**Discussion by pinkshirt-pinkshirt** [darthcloud/BlueRetro#854] [Q&A]
I've had success using the adapter with Saturn, Dreamcast and SNES. 
I also picked up a cable for the NES/Famicom and I can't get that one to work. 
The adapter powers up and the controller pairs like it does with all of the other consoles, but I can't control my NES or Famicom with it.
I noticed that if I disconnect the adapter from the cable, the console acts like P1 is pressing right+B
Based on that behavior and it working with all of the other consoles I was leaning toward a cable issue.
I'm wondering if anyone has any insight before I try to replace the cable.

Thanks

**Comment by darthcloud** [comment]
Likely bad cable if the SNES one work.
